build.gradle.kts

```diff
@@ -61,6 +61,7 @@ repositories {
 }
 
 plugins {
+    id("maven-publish")
```

Comment:
it is already defined on line 67

Reply:
Ah ok. My bad. I didnt realise this. Fixed